src/main/java/uk/gov/companieshouse/controller/SubmitController.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.companieshouse.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.exception.ServiceUnavailableException;
+import uk.gov.companieshouse.service.dissolution.chips.DissolutionChipsService;
+
+@RestController
+@RequestMapping("/dissolution-request/submit")
+public class SubmitController {
```

Comment:
Controller name sounds a bit generic. Can we rename it to "CHIPSSubmissionController" or something?

Reply:
Doesn't really match the endpoint tbh...